web/cobrands/greenwich/base.scss

```diff
@@ -6,6 +6,16 @@
 @import "../sass/base";
 @import "../sass/report_list_pins";
 
+body.frontpage {
+    color: $primary_text;
+    a,
+    .fake-link {
+        &:visited {
+            color: $links_visited;
+        }
+    }
+}
+
 #site-logo {
     background: url("/cobrands/greenwich/logo.gif") 0 50% no-repeat;
```

Comment:
I _think_ Greenwich has changed the shade of red they use in their logo since this GIF was made. We should really export a new version in a shade of red that matches their official site logo (`#C30025`).

And while we‘re at it, we could use the `svg-background-image()` mixin to serve up both a PNG and an SVG.

You’ll find a vector version of the Greenwich logo in the design drive, under `Projects > FixMyStreet Pro > fms-pro-customer-logos.sketch`.